artikel/2015-07-31-Hitzewelle-Sommerspaß-oder-Klimawandel.md

```diff
@@ -0,0 +1,34 @@
+#Hitzewelle: Sommerspaß oder Klimawandel?
+
+http://www.dw.com/de/hitzewelle-sommerspa%C3%9F-oder-klimawandel/a-18616694
+
+der Artikel "Hitzewelle: Sommerspaß oder Klimawandel?" spricht ein
+interessantes und wichtiges Thema an, weil wir diesen Sommer so viel 
+Hitzwelle in Deutschland gehabt haben.
+
+Der Artikel geht um Klimawandel und die Möglichkeit, ob es existiert. In 
+2015 gab es rekorde Temperaturen und häufiger Hitzwelle in Deutschland. 
+Im Juli 2015 gab es einen neuen Hitzrekord in Bayern mit 40,3 Grad Celsius 
+gemessen. Obwohl das heiße Wetter schön für Deutschen ist, finden die 
+meisten, dass ihre Wohnungen nicht für das heiße Wetter gut gebaut sind.
```

Comment:
I'd rather go for `gut geeignet`. `gut gebaut` refers to the _activity of building_ instead of the _state of building_.